testing/e2e/skr-tester/pkg/broker/client.go

```diff
@@ -150,28 +150,29 @@ func (c *BrokerClient) BuildRequestWithoutToken(payload interface{}, endpoint, v
 	return req, nil
 }
 
-func (c *BrokerClient) CallBroker(payload interface{}, endpoint, verb string) (map[string]interface{}, error) {
+func (c *BrokerClient) CallBroker(payload interface{}, endpoint, verb string) (map[string]interface{}, *int, error) {
```

Comment:
I would consider naming results (response, status code, error) to make it easier to read.
Three results is quite a bunch of values.

Reply:
Changed.